src/test/java/net/spy/memcached/CancelFailureModeTest.java

```diff
@@ -14,9 +14,9 @@ protected void setUp() throws Exception {
 
   @Override
   protected void tearDown() throws Exception {
-    // override teardown to avoid the flush phase
     serverList = ARCUS_HOST;
     client.shutdown();
+    super.tearDown();
```

Comment:
기존 주석이 아래와 같고, 의도는  flush phase를 피할려고 override한 것입니다.
- flush phase를 수행하여도 문제가 없는가요 ?
- super.tearDown()을 호출한다면, `client.shutdown()` 호출은 생략해도 될 듯 합니다.

```
// override teardown to avoid the flush phase
```

Reply:
flush phase를 수행하면 안되었던 이유는,
super.tearDown의 로직이 아래와 같기 때문입니다.
```
      client.shutdown();
      client = null;
      initClient();
      flushPause();
      assertTrue(client.flush().get());
      client.shutdown();
      client = null;
```
코드를 보면 client.shutdown 이후에 다시 client를 생성하여 flush를 합니다.
해당 테스트의 경우 존재하지 않는 캐시 서버에 접속하도록 하여 cancel을 유도하는 것인데
여기서 client 생성하여 flush 할 경우 cancel되어 테스트 오류가 발생합니다.
하지만 serverList = ARCUS_HOST; 라는 코드가 있기 때문에
shutdown이후 다시 client 생성하였을 때 정상적인 client를 생성하여 flush를 하는데 문제가 없습니다.
CancellationBaseCase 에서 serverList = ARCUS_HOST;가 들어간 이유도 이와 같습니다.

Reply:
이 부분이 super.tearDown 하도록 하니까 로컬에서는 오류가 발생 안했는데 
USE_ZK 상태에서 CI 오류가 발생하네요.
확인해보니 ARCUS_HOST로 무조건 serverList를 초기화한다음 재접속하니
ZK에 등록된 캐시 서버 중 ARCUS_HOST가 없으면 문제가 생깁니다.
어떻게 해결해야할지 고민해봐야겠네요.